crates/rolldown/tests/rolldown/topics/hmr/register_exports/_config.json

```diff
@@ -4,6 +4,5 @@
       "hmr": true
     }
   },
-  // TODO: should be executable
-  "expectExecuted": false
+  "hiddenRuntimeModule": false
```

Comment:
Any special reason to display runtime module for this test case, it looks a little verbose.

Reply:
Hide it now.